lib/modules/hosts/adultswim.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+import { Host } from '../../core/host';
+
+export default new Host('adultswim', {
+	name: 'Adult Swim',
+	domains: ['adultswim.com'],
+	logo: 'http://www.adultswim.com/favicon.ico',
+	detect: ({ href }) => (/^(?:https?)?:\/\/(?:(?:www)\.)?adultswim.com[\w\-\/:#]+videos\/([\S]+)/i).exec(href),
```

Comment:
Hm, might be better to use the following here:

`detect: ({ pathname }) => (/^\/videos\/([a-z0-9-#\/]+)$/i).exec(pathname)`